src/pose.rs

```diff
@@ -0,0 +1,85 @@
+use crate::{
+    nalgebra::{
+        base::{
+            dimension::{DimNameSum, U1},
+            MatrixN,
+        },
+        RealField,
+    },
+    nphysics::math::{Dim, Isometry, Point},
+};
+
+use specs::{Component, DenseVecStorage};
+
+/**
+Implement this for the Component type you want to sync transformation data to from the simulation.
+
+If an entity shares this Component and a [`BodyPartHandle`], [`PhysicsPoseSystem`] will call `sync`
+with the transform data of the Body part pointed at by the handle. Otherwise, if this Component is
+on an entity with a [`BodyComponent`], the first part on that Body (for non-multipart bodies, the
+Body itself) will be passed to `sync`. [`SimplePosition`] is provided as a simpl utility type which
+implements this trait.
+*/
+pub trait Pose<N: RealField>: Component + Send + Sync {
+    fn sync(&mut self, pose: &Isometry<N>);
+}
+
+// TODO: 64 bit implementation for amethyst
+#[cfg(all(feature = "amethyst", feature = "dim3"))]
+impl Pose<f32> for amethyst::core::Transform {
+    fn sync(&mut self, pose: &Isometry<f32>) {
+        *self.isometry_mut() = *pose;
+    }
+}
+
+#[cfg(all(feature = "amethyst", feature = "dim2"))]
+impl Pose<f32> for amethyst::core::Transform {
+    fn sync(&mut self, pose: &Isometry<N>) {
```

Comment:
I can't compile with ["dim2", "amethyst"]. Changing this line to 
```suggestion
    fn sync(&mut self, pose: &Isometry<f32>) {
```
seems to fix it.

Reply:
@cies caught this in https://github.com/distransient/specs-physics/pull/3/files#diff-31c13a9ef9b71e024d9f383cc35f390a

That change will get merged soon